CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # IdentityCache changelog
 
+#### Unreleased
```

Comment:
Maybe this should be `#### 0.6.0 Unreleased` to make it clear that the next release will have breaking changes.